### PR TITLE
Add command line flag for GPU tracing to ORBIT collector

### PR DESCRIPTION
**Pull request by ronaldfw**
This is also the first command line flag ever, so this adds parsing of command line flags via the absl::flags library. 

Please let me know if I should add other flags for tracing options, happy to do so.

### CODE REVIEW

File: OrbitLinuxTracing/TracerThread.cpp [hunk 1]
```diff
@@ -284,7 +288,7 @@ void TracerThread::Run(
   }
 
   bool gpu_event_open_errors = false;
-  if (trace_gpu_driver_events_) {
+  if (absl::GetFlag(FLAGS_trace_gpu_driver_events)) {
```
Comment (dimitry-) [inline OrbitLinuxTracing/TracerThread.cpp]: Should this be passed from main function? seems strange to rely on command-line flag outside of main function.

Comment (ronaldfw) [reply OrbitLinuxTracing/TracerThread.cpp]: That's how absl::flags work and how you can avoid a ton of plumbing. Every flag is a global that can be accessed from any .cpp file (if declared in the header file). IMO, the convenience of this this far outweighs the use of globals in this case.

Comment (beckerhe) [reply OrbitLinuxTracing/TracerThread.cpp]: Even though abseil supports this use case https://abseil.io/tips/45 argues strongly against accessing flags outside of main.

I personally have been burnt too often by the "flags-as-globals"-pattern, so I would argue against it, but maybe that's just me.

Comment (ronaldfw) [reply OrbitLinuxTracing/TracerThread.cpp]: That article also states: "Flags are appropriate in certain use cases, however: [...] Feature flags, when handled properly (and cleaned up afterward), are perfectly justified. "

I think this is such a case and will also not stay. We'll turn this on by default eventually, but I would like to give the option to turn it on when desired via a flag. 

Are you ok if I add a TODO to change this to either a configuration or remove entirely?

Comment (beckerhe) [reply OrbitLinuxTracing/TracerThread.cpp]: If it's temporary I'm absolutely fine with that!

Comment (ronaldfw) [reply OrbitLinuxTracing/TracerThread.cpp]: Ok, thanks, I'll still add a TODO to remove this.

Comment (ronaldfw) [reply OrbitLinuxTracing/TracerThread.cpp]: Just wanted to add that after considering this more thoroughly based on both of your comments, I definitely agree with not using flags throughout the code, only reading them in main, and then passing them to whatever places need them as a configuration (or whatever makes sense).

**Pull request merged**